routers/requests.py

```diff
@@ -0,0 +1,84 @@
+from typing import List
+from fastapi import APIRouter
+from schemas import requests as requests_schema
+
+router = APIRouter()
+
+
+@router.get("/requests", response_model=List[requests_schema.RequestInfo])
+async def get_requests(offset: int = 0, limit: int = 100):
```

Comment:
it is not needed if there is filter_requests

Reply:
Ok, will remove